src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    
+    public Ball getRandomBall() {
+        return new Ball(new ColorSupplier().getRandomColor(), new Random().nextInt(100));

```

Comment:
1) Again, point about creating Random. The same applies for colorSupplier
2) Don't forget about dealing with such numbers in the code (make them constants)
3) Please, pay attention to how nextInt() method works and what value you pass there